ScoutSuite/providers/aws/services/ec2/vpcs.py

```diff
@@ -0,0 +1,32 @@
+from ScoutSuite.providers.aws.resources.resources import AWSCompositeResources, AWSSimpleResources
+from ScoutSuite.providers.aws.facade.facade import AWSFacade
+from ScoutSuite.providers.aws.services.ec2.instances import EC2Instances
+from ScoutSuite.providers.aws.services.ec2.securitygroups import SecurityGroups
+from ScoutSuite.providers.aws.services.ec2.networkinterfaces import NetworkInterfaces
+
+
+class Vpcs(AWSCompositeResources, AWSSimpleResources):
```

Comment:
Ah, this is weird (but I understand why you're doing this).
Do you think we should merge the abstract classes `Resources` and `SimpleResources` (into `Resources`) to be more coherent? I think it would make sense but will require an other (hopefully final) wave of (small) changes. :face_with_head_bandage: This means `Resources` will define `fetch_all`, `get_resources_from_api` and `parse_resource` and thus `CompositeResources` will inherit `get_resources_from_api` and `parse_resource`. Would that make sense in the case of AWS? (It does with Azure.)

Reply:
Yea, I realized it's actually a case of diamond problem (if you were to call `super.fetch_all`, which parent's `fetch_all` should be called :|?)

Reply:
> Don't like the fact that `CompositeResources` inherits the `get_resources_from_api`, because it won't be used most of the time.

Not so sure about that, so far it could be used all the time since we have a call to `self.facade` in the `fetch_all` method of every classe that inherits from `CompositeResources`.
But surely we can postpone this conversation and wait for other cases where you would need to inherit from both `CompositeResources` and `SimpleResources`. :)